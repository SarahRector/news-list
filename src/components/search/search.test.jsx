import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Search from './search';

describe('Search component', () => {
  afterEach(() => cleanup());
  const testFunc = function() {};

  it('renders Search', () => {
    const { asFragment } = render(<Search
      search="Test"
      onSubmit={testFunc}
      onChange={testFunc}
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
