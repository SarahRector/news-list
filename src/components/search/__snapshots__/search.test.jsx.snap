// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders Search 1`] = `
<DocumentFragment>
  <label
    for="search"
  >
    News Search
  </label>
  <input
    id="search"
    name="search"
    type="search"
    value="Test"
  />
  <button
    data-testid="button"
  >
    Search For Articles
  </button>
</DocumentFragment>
`;
